espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/w3c/state/InputStateTable.java

```diff
@@ -27,11 +31,12 @@ public boolean hasInputState(String id) {
         return stateTable.containsKey(id);
     }
 
-    public synchronized static InputStateTable getInstance() {
-        if (globalInputStateTable == null) {
-            globalInputStateTable = new InputStateTable();
-        }
-        return globalInputStateTable;
+    public void addActionToCancel(ActionObject actionObject) {
+        cancelList.add(actionObject);
+    }
+
+    public List<ActionObject> getCancelList() {
+        return cancelList;
```

Comment:
should we return the list or a copy of it?

Reply:
Should be fine to use the actual list. This only gets accessed when the `/release` endpoint gets called, which is a single-threaded handler.

Reply:
Okay, yeah I can do that.